docs/releasing.md

```diff
@@ -1,39 +1,52 @@
 # Releasing
 
 To initiate a new production deployment:
+
 ```sh
 script/release vX.Y.Z
 ```
+
 See `script/release --help` for more information.
 
-> **Note:**  
-> Every production release will request an approval by the select few people before it can proceed.
+> [!NOTE]
+> Deployment workflow requires maintainer approval to run.
 
 What this does is:
+
 - Builds Linux binaries on Ubuntu;
 - Builds and signs Windows binaries on Windows;
 - Builds, signs, and notarizes macOS binaries on macOS;
 - Uploads all release artifacts to a new GitHub Release;
 - A new git tag `vX.Y.Z` is created in the remote repository;
 - The changelog is [generated from the list of merged pull requests](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes);
-- Updates cli.github.com with the contents of the new release;
+- Updates [GitHub CLI marketing site](https://cli.github.com) with the contents of the new release;
 - Updates the [`gh` Homebrew formula](https://github.com/williammartin/homebrew-core/blob/master/Formula/g/gh.rb) in the [`homebrew/homebrew-core` repo](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-core+%22gh%22+in%3Atitle&type=pullrequests).
 
+> [!NOTE]
+> `Homebrew/formulae.brew.sh` makes new formula versions available every 15 minutes through scheduled [CI workflow](https://github.com/Homebrew/formulae.brew.sh/actions/workflows/tests.yml).
```

Comment:
Since the issue was created, the previous workflow was refactored into this one.  Rather than nesting the content, just threw it into a separate note.